modules/stochastic_tools/examples/workshop/examples

```diff
@@ -1,49 +1,39 @@
 [Tests]
   [workshop]
-    issues = '#21242'
```

Comment:
you should keep that?

Reply:
The intention is that these examples will not be part of SQA.

Reply:
imo the tutorial is not just a random example, and the functional requirements make sense to me